src/main/kotlin/calculator/Operator.kt

```diff
@@ -0,0 +1,24 @@
+package calculator
+
+enum class Operator(val symbol: String) {
+    ADDITION("+") {
+        override fun calculate(first: Int, second: Int): Int = first.plus(second)
+    },
```

Comment:
```suggestion
enum class Operator(
    val symbol: String,
    val calculate: (Int, Int) -> Int,
) {
    ADDITION("+", { first, second -> first + second })
    ...
```
람다식을 프로퍼티로 갖고 있게 하는 것은 어떨까요?

Reply:
오 감사합니다!